web_development_101/web_development_frameworks/project_rails.md

```diff
@@ -23,6 +23,7 @@ Because you'll be doing so much stuff that we haven't taught you yet, this is a
 Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.  Please include your partner's github handle somewhere in the description if you had one and they would like attribution.
 
 * Add your solution below this line!
+ [SarfrazAnjum's solution](https://github.com/SarfrazAnjum/blogger2_v.1.0)
```

Comment:
@SarfrazAnjum Looks good, but can you add an asterisk `*` before your name to maintain the list format? Thanks!